src/components/application_manager/src/commands/mobile/set_global_properties_request.cc

```diff
@@ -152,22 +152,16 @@ void SetGlobalPropertiesRequest::Run() {
 
     params[strings::app_id] = app->app_id();
     SendUIRequest(params, true);
+
+    auto& help_prompt_manager = app->help_prompt_manager();
```

Comment:
@SNiukalov can we make it const?

Reply:
No using not constant method from object